common/src/test/java/org/conscrypt/javax/crypto/CipherTest.java

```diff
@@ -4377,7 +4385,11 @@ public void testAES_ECB_NoPadding_IncrementalUpdate_Success() throws Exception {
     }
 
     private void testAES_ECB_NoPadding_IncrementalUpdate_Success(String provider) throws Exception {
-        Cipher c = Cipher.getInstance("AES/ECB/NoPadding", provider);
+        String algorithm = "AES/ECB/NoPadding";
+        if (!isSupported(algorithm, provider)) {
```

Comment:
nit: assumeTrue(isSupported)?

Reply:
It is, but this is a helper method which is called in a loop over all providers from the actual test, so an assumption failure here would exit the whole test rather than skipping to the next provider. I suppose we could catch the relevant exception...

Reply:
Gonna submit this for now.  This test class is due an overhaul anyway, will consider using assumptions then.